examples/kokkos/02_simple_reduce.py

```diff
@@ -5,7 +5,7 @@
 class SquareSum:
     def __init__(self, n):
         self.N: int = n
-        self.total: int = 0
+        self.total: pk.double = 0
```

Comment:
Is there a larger integer type we could use instead?

Reply:
There's `pk.int64`, but it won't work here since for the style with `pk.main`, we pack all the reduction results into a single view and return that to Python. By default that view is of type `double`, so assigning an `int64_t` will result in a compilation error.